Formula/osm.rb

```diff
@@ -17,6 +17,7 @@ class Osm < Formula
 
   def install
     ENV["VERSION"] = "v"+version
+    ENV["BUILD_DATE"] = "$$(date +%Y-%m-%d-%H:%M)"
```

Comment:
Won't this set the environment variable `BUILD_DATE` to the string `$$(date +%Y-%m-%d-%H:%M)`?

Reply:
Yes, that string is used on the command line during `go build` where it will be evaluated:

```
% BUILD_DATE='$$(date +%Y-%m-%d-%H:%M)' make -n build-osm
...
go build -v -o ./bin/osm -ldflags "-X github.com/openservicemesh/osm/pkg/version.BuildDate=$(date +%Y-%m-%d-%H:%M) ...
```

Reply:
I see. If you don't mind, let's try to make this consistent with other fomulae that collect the build date/time from the environment:
```suggestion
    ENV["BUILD_DATE"] = Time.now.utc.strftime("%Y-%m-%d-%H:%M")
```
This should give exactly the same output as calling `date +%Y-%m-%d-%H:%M` from the shell.

Reply:
Makes sense, just changed it.